packages/next/client/image.tsx

```diff
@@ -40,13 +40,23 @@ type ImageConfig = ImageConfigComplete & { allSizes: number[] }
 export type ImageLoader = (resolverProps: ImageLoaderProps) => string
 
 export type ImageLoaderProps = {
-  config: Readonly<ImageConfig>
+  config?: Readonly<ImageConfig>
```

Comment:
Should this be included on the type if it's being exported, would it be better to exclude here and have a separate internal type that has `config`?

Reply:
@ijjk Good point! I updated the types in b41388d so this won't be exported 👍